.gitignore

```diff
@@ -11,7 +11,7 @@
 /coverage
 
 # Dependencies
-/node_modules
+node_modules
```

Comment:
How did it happen to you that there were multiple node_modules folders present?

Reply:
The command `pnpm install` will install all `node_modules` of all packages in the workspace.

Reply:
Ah right, I see I made a configuration wich prevented this from happening but could be invalid.